docs/index.html

```diff
@@ -8,7 +9,97 @@
     <link rel="icon" type="image/x-icon" href="favicon.ico">
     <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600" rel="stylesheet">
```

Comment:
Curious if we still need this `<link>` to Open Sans now that you've done all the awesome work in #104?